src/AccountProfile.jsx

```diff
@@ -82,6 +87,14 @@ const AccountProfile = (
       />
     </Avatar>
 
+    {verifications && (<VerifiedBadge>
+        <svg width="20" height="20" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's use ph icon instead `svg`:
```suggestion
        <i class="ph ph-seal-check" />
```